packages/form-js-editor/src/features/properties-panel/entries/LabelEntry.js

```diff
@@ -157,4 +162,20 @@ function TimeLabel(props) {
     setValue,
     variables
   });
+}
+
+// helpers //////////
+
+function getLabelText(field) {
+  const { type } = field;
+
+  if (type === 'group') {
+    return 'Group label';
+  }
+
+  if (type === 'iframe') {
```

Comment:
Not sure this is super necessary to change the wording here, I presume there were discussions around this?

Reply:
We kinda misused this one, because technically it's not only a label but also the accessible iFrame title. So we decided to use the existing label for it as it works for both.